### ISSUE TITLE
[registry-facade] pulls twice on cache miss

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/gitpod#10189]
### Bug description

When we pull from US, this does so from EU. Therefore, pulling twice due to a cache miss (and missing cache often) becomes expensive.

### Steps to reproduce

This could be related a cache miss for a `workspace-image` (perhaps because we pushed new base images), or a cache miss on a newly built workspace image (for which we just built and pushed via image-builder-mk3). 

### Expected behavior

@aledbf were you able to socialize possible options with @csweichel for this last week? It's okay to say no, I know you two were working the S3 bucket issue instead.

### Anything else?

Need analysis and design work, not ready to be picked up.

**Comment by aledbf** [comment]
> aspect cost efficiency

Technical aspects should drive this change. The IPFS feature was added a long time after the component, and the additional interzonal costs were a side effect of the IPFS implmentation.

**Comment by aledbf** [comment]
@kylos101 we need this change

**Comment by kylos101** [comment]
Thank you for the heads up, @aledbf !

Can you share why? I assume it is to achieve faster start times, avoiding duplicate image pull.

I ask because this issue was closed by stalebot, and is not in our groundwork.

Also, I assume it is unrelated to [this issue](https://github.com/gitpod-io/gitpod/issues/13915) (which is in our groundwork).

**Comment by aledbf** [comment]
> Also, I assume it is unrelated to https://github.com/gitpod-io/gitpod/issues/13915 (which is in our groundwork).

No. This is technical debt and cleanup since the introduction of IPFS.
